.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,45 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
```

Comment:
Maybe it's not needed for tags as it happens in branches and pull requests?

Reply:
I think you are right. I will change it.